### DISCUSSION TITLE
あなたの使っているスマートフォンを教えてTell us what smartphone you use.

### DISCUSSION THREAD

**Discussion by hatodove22** [NAIST-SE/AttractingContributors#517] [General]
https://github.com/hatodove22/Your-Phone.git

あなたの使っているスマホを投稿してください．

**Comment by ani-hovhannisyan** [comment]
Hi, I have added 8th in your project, please can you do it back to mine?
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
Project https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle.git

**Reply by hatodove22** [reply]
Of course! I've already done it.
